Common/src/Pollster/TaskHandler.cs

```diff
@@ -357,6 +357,8 @@ public async Task<bool> AcquireTask()
           taskData_ = await taskTable_.ReadTaskAsync(messageHandler_.TaskId,
                                                      CancellationToken.None)
                                       .ConfigureAwait(false);
+          logger_.LogInformation("Task is not running on the other polling agent, status : {status}",
```

Comment:
Does this log belong to the info status? I am not seeing the value for the people which will operate.

Reply:
I agree.

Reply:
And this log should not happen very often.